roles/etcd_migrate/tasks/migrate.yml

```diff
@@ -42,6 +44,7 @@
     --lease-duration 1h
   environment:
     ETCDCTL_API: 3
+    PATH: "/usr/local/bin:/var/usrlocal/bin:{{ ansible_env.PATH }}"
```

Comment:
Perhaps, create a separate PR that extends the `openshift_facts` module and sets the `openshift.common.admin_binary` to the proper location. @ashcrow when does the `/var/usrlocal/bin` applies and when `/usr/local/bin`? Is there any rule to that? Optionally, the module could check if `/var/usrlocal/bin/oadm` or `/usr/local/bin/oadm` exists and then set the `admin_binary` variable.